### PR TITLE
fix: csr favicon & searchbar

### PR DESCRIPTION
**Pull request by naren-bellala**
#### 📲 What

A description of the change.
Task 1827: Fix favicon

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/templates/src/csr/public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,44 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "name": "Amido Stacks",
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/csr/public/manifest.json:2]: When I build this locally, I still see the title as `React App`. Might be worth changing?

File: packages/scaffolding-cli/templates/src/csr/src/components/Search/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import {render} from "@testing-library/react"
+import React from "react"
+import {Search} from "."
+
+test("renders ApiPane snapshot", () => {
+    const {asFragment} = render(<Search getSearchResults />)
+    expect(asFragment()).toMatchSnapshot()
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/csr/src/components/Search/index.test.tsx:7]: The tests aren't running? And the snapshsot not checked in? Still see old Facebook tests.

**Comment by estherthetesteramido** [comment]
The package.json has a lot of depenedencies rather than devDependencies. Are these all needed? Jest at run time?

File: packages/scaffolding-cli/templates/src/csr/src/components/Search/index.test.tsx (hunk 2 above)
Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/csr/src/components/Search/index.test.tsx:7]: I still have not touches any testing files. As we have an outstanding task on the board. @estherthetesteramido

File: packages/scaffolding-cli/templates/src/csr/public/manifest.json (hunk 1 above)
Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/csr/public/manifest.json:2]: The title of the application on the browser is driven from the index.html and not from manifest. I guess the manifest dictates the PWA or mobile app icon etc. I will fix the title soon.

**Comment by naren-bellala** [comment]
> The package.json has a lot of depenedencies rather than devDependencies. Are these all needed? Jest at run time?

The package.json is getting tidied up iteratively when I start working on relevant task.  By default all _Create react app_ dependencies are run-time deps. I have just moved the lint and typescript ones out soon I will move the rest.

File: packages/scaffolding-cli/templates/src/csr/src/components/Search/index.test.tsx (hunk 2 above)
Comment (estherthetesteramido) [reply packages/scaffolding-cli/templates/src/csr/src/components/Search/index.test.tsx:7]: Fair. Can we instead then delete the files if they aren't being used?

**Comment by estherthetesteramido** [comment]
> > The package.json has a lot of depenedencies rather than devDependencies. Are these all needed? Jest at run time?
> 
> The package.json is getting tidied up iteratively when I start working on relevant task. By default all _Create react app_ dependencies are run-time deps. I have just moved the lint and typescript ones out soon I will move the rest.

Created AB#1824 to capture this.

File: packages/scaffolding-cli/templates/src/csr/public/manifest.json (hunk 1 above)
Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/csr/public/manifest.json:2]: **Sorted now. Try it now**

**Pull request merged**